src/main/kotlin/uk/gov/gdx/datashare/config/OpenApiConfiguration.kt

```diff
@@ -30,11 +32,27 @@ class OpenApiConfiguration(
         Server().url("/").description("Current url"),
       ),
     )
+    .tags(
+      listOf(
+        Tag().name("01. Acquirer").description("Acquiring events from the platform"),
+        Tag().name("02. Supplier").description("Supplying events to the platform"),
+        Tag().name("11. Acquirers").description("Maintenance of Acquirers"),
+        Tag().name("12. Suppliers").description("Maintenance of Suppliers"),
+        Tag().name("13. Admin").description("Admin functions"),
+        Tag().name("20. Data").description("Maintenance of Data"),
+      ),
+    )
     .info(
       Info().title("GDX Data Share API")
         .version(version)
+        .license(License().name("MIT").url("https://opensource.org/license/mit-0"))
         .description("API for obtaining data about citizen life events")
-        .contact(Contact().name("GDX Vision Team").email("mike.willis@digital.cabinet-office.gov.uk")),
+        .contact(
+          Contact()
+            .name("GDX Platform Team")
+            .email("gdx-dev-team@digital.cabinet-office.gov.uk")
```

Comment:
Does this address work externally yet?

Reply:
I don't think so, I'm gonna poke Farrah again soon